tools/mkimage-raw-efi/patches/0001-TPM-build-issue-fixing.patch

```diff
@@ -0,0 +1,72 @@
+From 617b08377dbaa9ea3876b5585fe0ba36286fbed6 Mon Sep 17 00:00:00 2001
+From: Dennis Chen <dennis.chen@arm.com>
```

Comment:
Any chance we can upstream this? Is there a PR? Its annoying duplicating it...

Reply:
Not sure. I sort of took it as is from @arm64b

Reply:
I think we should take it as is for now. As we discussed yesterday, our effort probably would be better placed in getting a proper bootable kernel image with initrd built right in, and then the whole grub piece goes away, both for `raw-efi` and `iso-efi`.